### DISCUSSION TITLE
Is there any information about new PYBD base on Ambiq Apollo 4+?

### DISCUSSION THREAD

**Discussion by sam0910** [micropython/micropython#11475] [Other]
Greetings!

I found an [interesting link](https://micropython.org/resources/MicroPython10YearsPoster.pdf) about new PYBD base on Ambiq Apollo 4,
It said that it's power consumption of only 5mA in full power model. 
That is so interesting information for wearable and battery powered device.
And it seems that supports MIPI display with AMOLED, and bunch of things for wearables.
- 192 MHz clock frequency
- Floating Point Unit (FPU), Memory Protection Unit (MPU)
- BLE 5.1, 2 Mbps, extended advertising packets
- 2D/2.5D GPU
- MIPI DSI 1.2 up to 500 Mbps, up to 454x454 resolution
- Audio processing etc..

The MCU is 4.7mm BGA packages and seems a total gift box for IOT and wearable devices. :)

Does anyone know about any information?

**Comment by gretel** [comment]
be cool if the 20 years poster gets clickable links to have some references. regards
